src/main/webapp/web.xml

```diff
@@ -6,9 +6,19 @@
     <servlet>
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
+        
     </servlet>
     <servlet-mapping>
         <servlet-name>index</servlet-name>
         <url-pattern>/index</url-pattern>
     </servlet-mapping>
+    
+    <servlet>
+        <servlet-name>GetAllDriver</servlet-name>
+        <servlet-class>mate.controller.driver.GetAllDriverController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>GetAllDriver</servlet-name>
+        <url-pattern>/drivers/all</url-pattern>
+    </servlet-mapping>
 </web-app>
```

Comment:
let's make the code more consistent and use @WebServlet annotation for all controllers(or you can configure everything via web.xml)